powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/fido2/PowerAuthAssertionProvider.java

```diff
@@ -96,7 +94,7 @@ public AssertionChallenge provideChallengeForAssertion(List<String> applicationI
 
     @Override
     @Transactional
-    public AssertionChallenge approveAssertion(String challengeValue, AuthenticatorDetail authenticatorDetail) throws Fido2AuthenticationFailedException {
+    public AssertionChallenge approveAssertion(String challengeValue, AuthenticatorDetail authenticatorDetail, AuthenticatorData authenticatorData, CollectedClientData clientDataJSON) throws Fido2AuthenticationFailedException {
```

Comment:
A quite long parameter list. I prefer a parameter object

Reply:
Let's leave this as is for now, we can fix this in a separate ticket.